toolkit/imageconfigs/scripts/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -eux
+
+# cleanup symlinks created by the toolkit that are not needed for base images
```

Comment:
You should add a code comment saying why this is needed. Considering that this symlink is being added by the `filesystem` package, this seems like intentional behavior for Mariner. So, there should be justification for why the bare-metal and qemu-guest images need to be different.

Also, add this comment to the PR description as well.

Reply:
updated from a logged bug.